README.md

```diff
@@ -62,6 +64,35 @@ should not require general ternary matches on the destination address; the
 constraint documents this intend and let's us catch accidental ternary matches
 installed by the control plane at runtime.
 
+## API
+
+At a high level, p4-constraint's API consists of only two functions:
```

Comment:
Maybe we should end this section with: There are also other APIs for parsing, etc, but those are only needed for projects building on top of p4-constraints, not simple users of the library? This makes it sound like there is nothing else?

Reply:
Yes, good point

Reply:
Done.